### ISSUE TITLE
tesst to be deleted

### ISSUE THREAD

**Issue by tay1orjones** [carbon-design-system/carbon#20295]
when a blank issue has something like Databand.ai or chrome or JAWS and it's Severity 2, some labels should be applied
